dist/s_tags

```diff
@@ -1,41 +1,37 @@
 #! /bin/sh
 
-# Build tags file.
-#
-t=__a.c
-trap 'rm -f $t; exit 0' 0 1 2 3 13 15
+# Build tags files.
+trap 'rm -f tags; exit 0' 0 1 2 3 13 15
 
```

Comment:
I thought of that too, but currently this script can't be run from anywhere other than `dist` (it assumes there are `../src/include` and a `../build_posix/` directories). I agree it might be a little late at that point, though, I'll put in a quick test to ensure we're where we think we are.